x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/AbstractPhysicalOperationProviders.java

```diff
@@ -35,7 +35,7 @@
 import java.util.Set;
 import java.util.function.Consumer;
 
-abstract class AbstractPhysicalOperationProviders implements PhysicalOperationProviders {
+public abstract class AbstractPhysicalOperationProviders implements PhysicalOperationProviders {
```

Comment:
Opened this class up to get access to intermediateAttributes - this is a great candidate for future refactorings.